portfolio/pom.xml

```diff
@@ -23,6 +23,26 @@
       <version>1.9.59</version>
     </dependency>
     
+    <dependency>
+        <groupId>com.google.appengine</groupId>
+        <artifactId>appengine-testing</artifactId>
+        <version>1.5.0</version>
```

Comment:
All of these versions should match you sdk version: 1.9.59. I think that's why you're seeing classpath problems.

Reply:
ah ok thanks! I'll try that :)

Reply:
that worked!!